baselines/exo.md

```diff
@@ -239,125 +268,165 @@ An agency point of contact SHOULD be included for aggregate and/or failure repor
   Exchange Online \| Microsoft
-  Docs](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission)
+  Learn](https://learn.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission)
 
 - [Use the Microsoft 365 admin center to enable or disable SMTP AUTH on
   specific mailboxes \| Microsoft
-  Docs](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#use-the-microsoft-365-admin-center-to-enable-or-disable-smtp-auth-on-specific-mailboxes)
+  Learn](https://learn.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#use-the-microsoft-365-admin-center-to-enable-or-disable-smtp-auth-on-specific-mailboxes)
 
 ### License Requirements
 
 - N/A
 
 ### Implementation
 
-SMTP AUTH can only be disabled tenant-wide using Exchange Online
-PowerShell. Follow the instructions listed at [Disable SMTP AUTH in your
-organization \| Microsoft
-Docs](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#disable-smtp-auth-in-your-organization).
+#### MS.EXO.5.1v1 instructions:
+SMTP AUTH can be disabled tenant-wide using the Exchange admin center
+or Exchange Online PowerShell. Follow the instructions listed at [Disable
+SMTP AUTH in your organization \| Microsoft
+Learn](https://learn.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#disable-smtp-auth-in-your-organization).
 
+#### MS.EXO.5.2v1 instructions:
 To enable SMTP AUTH on a per-mailbox basis, follow the instructions
 listed at [Use the Microsoft 365 admin center to enable or disable SMTP
 AUTH on specific mailboxes \| Microsoft
-Docs](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#use-the-microsoft-365-admin-center-to-enable-or-disable-smtp-auth-on-specific-mailboxes).
+Learn](https://learn.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/authenticated-client-smtp-submission#use-the-microsoft-365-admin-center-to-enable-or-disable-smtp-auth-on-specific-mailboxes).
 
 ## 6. Calendar and Contact Sharing
 
 Exchange Online allows the creation of sharing polices that soften
 default restrictions on contact and calendar details sharing. These
-policies should only be enabled with caution and must comply with the
-following policies.
+policies should only be enabled with caution and after considering
+the following policies.
 
 ### Policies
 
 #### MS.EXO.6.1v1
-Contact folders SHALL NOT be shared with all domains, although they
-  MAY be shared with specific domains.
+Contact folders SHALL NOT be shared with all domains
+
 - _Rationale:_ TODO
 - _Last modified:_ June 2023
+- Note: Contact folders MAY be shared with specific domains
```

Comment:
I would use _Note:_ here. To keep everything consistent for Policies section. (also other documents)